src/bundle/Resources/views/themes/admin/content/form_fields.html.twig

```diff
@@ -42,25 +42,29 @@
 {%- endblock %}
 
 {% block form_row_subfield -%}
-    {% set wrapper_class = 'ez-data-source__field ez-data-source__field--' ~ name %}
+    {% set wrapper_class = 'ibexa-data-source__field ibexa-data-source__field--' ~ name ~ ' ez-data-source__field ez-data-source__field--' ~ name %}
```

Comment:
Why `ez-` classes were left? For fields, which are not converted to `ibexa-` yet?

Reply:
Exactly, otherwise all css and js files would have to be adjusted, these classes would have to be removed at the end of refactoring